editor/components/colors/with-colors.js

```diff
@@ -1,91 +1,134 @@
 /**
  * External dependencies
  */
-import memoize from 'memize';
-import { get } from 'lodash';
+import { find, get, isFunction, isString, kebabCase, reduce, upperFirst } from 'lodash';
 
 /**
  * WordPress dependencies
  */
 import { createHigherOrderComponent, Component, compose } from '@wordpress/element';
 import { withSelect } from '@wordpress/data';
+import { deprecated } from '@wordpress/utils';
 
 /**
  * Internal dependencies
  */
-import { getColorValue, getColorClass, setColorValue } from './utils';
+import { getColorValue, getColorClass } from './utils';
+import { default as withColorsDeprecated } from './with-colors-deprecated';
```

Comment:
Why is this `import { default as withColorsDeprecated } from './with-colors-deprecated';` instead of just `import withColorsDeprecated from './with-colors-deprecated';`?